helm_deploy/hmpps-resettlement-passport-ui/files/grafana-dashboards/hmpps-resettlement-passport-dashboard.json

```diff
@@ -602,8 +590,8 @@
       {
         "current": {
           "selected": false,
-          "text": "hmpps-resettlement-passport-dev",
-          "value": "hmpps-resettlement-passport-dev"
+          "text": "hmpps-resettlement-passport-prod",
+          "value": "hmpps-resettlement-passport-prod"
```

Comment:
Default probably should be dev

Reply:
not sure about this, the data we need to report on is prod, and we want to open this up to the wider team

Reply:
Good point - let's point it to prod then.